a10_neutron_lbaas/v2/handler_hm.py

```diff
@@ -38,41 +34,39 @@ def _set(self, c, set_method, context, hm):
             url = hm.url_path
             expect_code = hm.expected_codes
 
-        set_method(hm_name, hm_map[hm.type],
-                   hm.delay, hm.timeout, hm.max_retries,
-                   method=method, url=url, expect_code=expect_code)
+        args = self.meta(hm, 'hm', {})
 
-    def _create(self, c, context, hm):
-        self._set(c, c.client.slb.hm.create, context, hm)
-
-        if hm.pool:
-            c.client.slb.service_group.update(
-                hm.pool.id,
-                health_monitor=self._hm_name(hm))
+        set_method(hm_name, a10_os.hm_type(c, hm.type),
+                   hm.delay, hm.timeout, hm.max_retries,
+                   method=method, url=url, expect_code=expect_code,
+                   axapi_args=args)
 
     def create(self, context, hm):
         with a10.A10WriteStatusContext(self, context, hm) as c:
-            self._create(c, context, hm)
+            try:
+                self._set(c, c.client.slb.hm.create, context, hm)
+            except acos_errors.Exists:
+                pass
```

Comment:
More unlogged exceptions.  I can see why we'd want to just go ahead when it exists because that's not a potentially fatal error.

Are we concerned about other exceptions crashing Neutron?  This has been the behavior I've witnessed.  Not sure if this is a case where we WANT Neutron to crash if something fatal occurs.  You know better than me. :)